src/main/java/mekanism/common/registries/MekanismBlockTypes.java

```diff
@@ -317,8 +318,14 @@ private MekanismBlockTypes() {
           .withEnergyConfig(() -> TELEPORTER_USAGE, MekanismConfig.storage.teleporter)
           .withSupportedUpgrades(EnumSet.of(Upgrade.ANCHOR))
           .without(AttributeStateActive.class, AttributeStateFacing.class, AttributeParticleFX.class)
+          .with(new AttributeSecurity(), new AttributeMultiblock())
```

Comment:
You should rebase as I made changes earlier today to make it so that these attributes only ever have one instance created to save slightly on memory.